app/javascript/components/subject/MoveToHousehold.js

```diff
@@ -73,50 +205,82 @@ class MoveToHousehold extends React.Component {
           console.error(error);
         });
     });
+  };
+
+  /**
+   * Grabs a formatted string with the name of the patient who is being moved to a household.
+   */
+  getPatientName = () => {
+    return `${this.props.patient?.last_name || ''}, ${this.props.patient?.first_name || ''} ${this.props.patient?.middle_name || ''}`;
+  };
+
+  /**
+   * Handles a key press event on the search form control.
+   * Checks for enter button press and prevents submisson event.
+   * @param {Object} event
+   */
+  handleKeyPress(event) {
+    if (event.which === 13) {
+      event.preventDefault();
+    }
   }
 
-  createModal(title, toggle, submit) {
+  createModal(title, toggle) {
     return (
-      <Modal size="lg" show centered onHide={toggle}>
+      <Modal dialogClassName="modal-move-household" show centered onHide={toggle}>
```

Comment:
Thoughts on maybe making this modal size a little bit bigger? Maybe not as big as the import one, but big enough so that the pagination options on the bottom would show up on one row? 

<img width="1194" alt="Screen Shot 2020-11-19 at 12 21 45 PM" src="https://user-images.githubusercontent.com/9956561/99700960-d8a7c500-2a61-11eb-8e22-9094dc97acad.png">

Reply:
Agreed, done.